test/core/slice/BUILD

```diff
@@ -20,6 +20,26 @@ licenses(["notice"])  # Apache v2
 
 load("//test/core/util:grpc_fuzzer.bzl", "grpc_fuzzer")
 
+grpc_fuzzer(
+    name = "b64_encode_fuzzer",
```

Comment:
The fuzzer tests are currently failing on windows for compilation reasons.

Please add
tags = ["no_windows"],
to each of the fuzzer build targets to resolve this